src/main/java/com/autotune/common/k8sObjects/KruizeObject.java

```diff
@@ -226,8 +232,17 @@ public void setValidationData(ValidationResultData validationData) {
         this.validationData = validationData;
     }
 
+    public String getClusterName() {
+        return clusterName;
+    }
+
     @Override
     public String toString() {
+        // Creating a temparory cluster name as we allow null for cluster name now
+        // Please change it to use `clusterName` variable itself if there is a null check already in place for that
+        String tmpClusterName = "";
+        if (clusterName != null)
+            tmpClusterName = clusterName;
```

Comment:
Why is this a special case? `clusterName` = "null" should be handled by the same regular code itself

Reply:
Thanks for your review @dinogun  As there was no `null` check in place for `clusterName` I was substituting empty string if the cluster name is `null` to avoid null be printed as cluster name if kruize object is printed.

It also contradicts with a cluster named as `NULL` if we allow printing null as cluster name if it's not set